package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "peggy",
+  "name": "peggy-monorepo",
```

Comment:
This name change will cause havoc in `npm`.  Many downstream tools believe that the repo name and the listing name must match.

Reply:
Example please?  `node-cbor` is a package that has this setup, and it seems to have been working ok.  Granted, it's not terribly popular compared to `peg.js`.

Reply:
rollup 1.x

Reply:
I didn't follow the discussion they had.  Do you have a link, please?

Reply:
i'm not aware of a discussion around this.  it's just something i went through years ago